changelog.d/5-internal/ldap-scim-bridge

```diff
@@ -0,0 +1 @@
+Upgraded ldap-scim-bridge helm chart to use tag 0.9 for container image
```

Comment:
could you also fix the link in the docs that you showed me the other day?  the one pointing to temp-jun-something?

Reply:
Yes, that part of doc is removed - https://github.com/wireapp/wire-server-deploy/pull/703